src/bugmon/utils.py

```diff
@@ -117,9 +116,8 @@ def download_zip_archive(url: str) -> Generator[Path, None, None]:
 
 def is_pernosco_available() -> bool:
     """Determines if pernosco-submit is properly configured"""
-    py_path = Path(sysconfig.get_path("platlib"))
-    pernosco_shared = py_path / "pernoscoshared"
-    return PERNOSCO is not None and pernosco_shared.exists()
+    result = subprocess.run(["pernosco-submit", "--help"], check=False)
```

Comment:
Do you want to redirect to dev/null or do you want to see the help output on success?